rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -46,8 +46,24 @@ static void set_rsx_dmactl(rsx::thread* render, u64 get_put)
 		render->fifo_ctrl->abort();
 
 		// Unconditional set
-		while (!render->new_get_put.compare_and_swap_test(u64{umax}, get_put))
+		while (true)
 		{
+			if (render->new_get_put.compare_and_swap_test(u64{umax}, get_put))
+			{
+				break;
+			}
```

Comment:
I don't see a reason for this change

Reply:
What if the RSX stopped (by Emu.Kill()) before the previous new_get_put request compelted, wouldn't that cause a an infinite loop if a new new_get_put request is made?

Reply:
I have no idea what that means. All I see is that you moved the condition into the loop, which seems like a random change that was made for no reason.

Reply:
Ah ok.

Reply:
Fixed.